TrafficCapture/trafficCaptureProxyServer/src/main/java/org/opensearch/migrations/trafficcapture/proxyserver/CaptureProxy.java

```diff
@@ -299,6 +345,9 @@ protected static SslContext loadBacksideSslContext(URI serverUri, boolean allowI
     }
 
     protected static Map<String, String> convertPairListToMap(List<String> list) {
+        if (list == null) {
```

Comment:
Should we remove this or change to an asset if it's unexpected case?

Reply:
What's the problem with keeping it?  Right now, JCommander doesn't return null when the arg isn't included.  I was concerned that if that ever changed, we'd get NPEs where we'd really just want to know that there weren't elements defined.

Reply:
i'm fine with keeping it in for now, would like this to be tested in the future